movement/watch_faces/complication/totp_face.c

```diff
@@ -6,24 +6,49 @@
 #include "TOTP.h"
 
 // Use https://cryptii.com/pipes/base32-to-hex to convert base32 to hex
-// Use https://totp.danhersam.com/ to generate test codes for verification
+// Use https://github.com/susam/mintotp to generate test codes for verification
 
-static const uint8_t num_keys = 2;
+static const uint8_t num_keys = 6;
```

Comment:
No need to make this change yourself, but before merging I'm going to restore this to just two keys, and add some notes above and below signaling to folks that this is where they put their key data and what the different definitions mean.

Reply:
Alright, i'll let you do the modification then :)